pkg/kubelet/winstats/perfcounters.go

```diff
@@ -94,16 +99,17 @@ func (p *perfCounter) getData() (uint64, error) {
 		return 0, fmt.Errorf("unable to collect data from counter. Error code is %x", ret)
 	}
 
+	data = 0
 	for i := 0; i < int(bufCount); i++ {
 		c := filledBuf[i]
-		data = uint64(c.FmtValue.DoubleValue)
+		data += uint64(c.FmtValue.DoubleValue)
```

Comment:
This also looks like a bug

Reply:
Indeed, it looks like a bug. However, it didn't really cause an issue. ``getData`` was only called for the ``\Processor(_Total)\% Processor Time``, ``\Process(_Total)\Working Set - Private``, ``\Memory\Committed Bytes``, which only ever had one element in the array.

This was failing for the networking perfcounters, which would have more than 1 elements in the array, including NICs that would have 0 traffic on them, resulting in the ``TestPerfCounter`` test failing. For network perfcounters, ``getDataList`` is used instead, which is a map between the NIC name and the value associated with that perfcounter. In this case, if ``getData`` is called, I think it's reasonable to return the overall sum over all NICs.

In any case, this doesn't change the current functionality of winstats.